13_iv_antibiotics_pharmacy.sql

```diff
@@ -0,0 +1,87 @@
+CREATE OR REPLACE TABLE `physionet.iv_abx_pharmacy` as
+
+select 
+subject_id,
+hadm_id,
+stay_id,
+DATETIME_TRUNC(starttime, HOUR) as chart_hour,
+intime,
+outtime,
+medication,
+min(entertime) as entertime,
+min(starttime) as starttime,
+min(route) as route,
+max(stoptime) as stoptime,
+min(frequency) as frequency,
+min(disp_sched) as disp_sched,
+min(doses_per_24_hrs) as doses_per_24_hrs,
```

Comment:
My concern is that, since this query is for constructing a table, that there are now two ways to interpret some of the aggregate columns. Some of these, like `min(doses_per_24_hours)`, may not have to be aggregated, and others, like `disp_sched`, may mean something in the new table that they didn't in the old table.

For the columns that absolutely need to be kept, would it make sense to rename them to, say, `min_disp_sched` for `min(disp_sched)`?

Reply:
I agree, the minimum is a stopgap - rafiya and I discussed that in the future, we might want to consider taking the latest entry for a given medication based on entertime. For most entries (when grouped on stay_id, chart_hour, medication), the aggregated fields are the same across all grouped records, but they might differ if the pharmacy receives several of the same order (presumably because the clinician had to revise their order).

Reply:
also atm none of these fields are being used in our final schema, they're just there for potential context in the future, so I don't think it would hurt to rename them anyway